### PR TITLE
Updated tests to work after Washington D.C. SNOW UPDATE

### PR DESCRIPTION
**Pull request by lfcabrera14**
Updated tests to work after Washington D.C. SNOW UPDATE

### CODE REVIEW

File: src/test/java/appsCommon/Pages/Playwright_NativeView_Dashboard_Page.java [hunk 1]
```diff
@@ -5,7 +5,10 @@ public class Playwright_NativeView_Dashboard_Page {
     /**
      * PROFILE BUTTON
      */
-    public static String profileButton = "Avatar: available, user";
+    //**** for test env ***
+//    public static String profileButton = "Avatar: available, user";
```
Comment (hamidnesar) [inline src/test/java/appsCommon/Pages/Playwright_NativeView_Dashboard_Page.java:9]: looks like this page object is also created on line 11, do you still need to have this commented? 
I see one is for test env and one for sandbox env

File: src/test/java/appsCommon/PlaywrightUtils/Playwright_ServiceNow_Common_Methods.java [hunk 2]
```diff
@@ -37,7 +37,10 @@ public static void side_Door_Test_Account_Login_Impersonate(String name){
         page.waitForLoadState();
         page.reload();
         MiscUtils.sleep(2000);
-        page.getByLabel(Playwright_NativeView_Dashboard_Page.profileButton).click();
+        //**** for test env ***
+//        page.getByLabel(Playwright_NativeView_Dashboard_Page.profileButton).click();
```
Comment (hamidnesar) [inline src/test/java/appsCommon/PlaywrightUtils/Playwright_ServiceNow_Common_Methods.java:41]: Do you need to retain the commented line for test env?

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, 
Please review the comments. Thanks!

File: src/test/java/PlaywrightTests/steps/Example.java [hunk 3]
```diff
@@ -7,27 +7,39 @@
...
+//            page.getByRole(AriaRole.BUTTON, new Page.GetByRoleOptions().setName("Log in")).click();
+//            page.navigate("https://service-test.nci.nih.gov/now/nav/ui/classic/params/target/%24pa_dashboard.do");
+//            assertThat(page.getByLabel("StartNow Implementation", new Page.GetByLabelOptions().setExact(true))).containsText("StartNow Implementation");
+//            assertThat(page.getByLabel("All menu", new Page.GetByLabelOptions().setExact(true))).containsText("StartNow Homepage");
+//            page.getByPlaceholder("Filter").click();
+//            page.getByPlaceholder("Filter").fill("CHARMS");
+//            page.getByRole(AriaRole.LINK, new Page.GetByRoleOptions().setName("Dashboard")).click();
+//            Page page1 = page.waitForPopup(() -> {
+//                page.frameLocator("iframe[name=\"gsft_main\"]").getByLabel("Eligibility Review Needed, Total number of records is:").click();
+//            });
```
[the remark this answers is not in the document]
Comment (iamdez99) [reply src/test/java/PlaywrightTests/steps/Example.java:30]: Yes, please add this to the git ignore or just delete it and re-create it again if you need to

**Review by iamdez99 — CHANGES_REQUESTED** [review]
I agree with Hamids requests. Please make requested changes.

File: src/test/java/PlaywrightTests/steps/Example.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (lfcabrera14) [reply src/test/java/PlaywrightTests/steps/Example.java:30]: I'll add this to the git ignore, thanks

File: src/test/java/PlaywrightTests/steps/Example.java
Comment (matakevin) [inline src/test/java/PlaywrightTests/steps/Example.java:1]: Seems this class is still present.  Please delete and push changes.

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/DEPRECATED TESTS.feature
Comment (matakevin) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/DEPRECATED TESTS.feature:1]: This class seems to be missing some tags for all the scenarios.  Seems like its just the user story or test scenario.

File: src/test/java/appsCommon/Pages/Playwright_NativeView_Dashboard_Page.java (hunk 1 above)
Comment (matakevin) [reply src/test/java/appsCommon/Pages/Playwright_NativeView_Dashboard_Page.java:9]: I second Hamid's comments

File: src/test/java/appsCommon/PlaywrightUtils/Playwright_ServiceNow_Common_Methods.java (hunk 2 above)
Comment (matakevin) [reply src/test/java/appsCommon/PlaywrightUtils/Playwright_ServiceNow_Common_Methods.java:41]: Seconded

**Review by matakevin — CHANGES_REQUESTED** [review]
Please review requested changes

File: src/test/java/PlaywrightTests/steps/Example.java
Comment (lfcabrera14) [reply src/test/java/PlaywrightTests/steps/Example.java:1]: just did

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/DEPRECATED TESTS.feature
Comment (lfcabrera14) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/selenium/Features/DEPRECATED TESTS.feature:1]: yes I just left the original tag that was on it before I moved it to the deprecated test case folder

File: src/test/java/PlaywrightTests/steps/Example.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (lfcabrera14) [reply src/test/java/PlaywrightTests/steps/Example.java]: just deleted this class and folder

File: src/test/java/appsCommon/Pages/Playwright_NativeView_Dashboard_Page.java (hunk 1 above)
Comment (lfcabrera14) [reply src/test/java/appsCommon/Pages/Playwright_NativeView_Dashboard_Page.java]: done

File: src/test/java/appsCommon/PlaywrightUtils/Playwright_ServiceNow_Common_Methods.java (hunk 2 above)
Comment (lfcabrera14) [reply src/test/java/appsCommon/PlaywrightUtils/Playwright_ServiceNow_Common_Methods.java]: done

**Review by matakevin — APPROVED** [review]
LGTM

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved

**Review by hamidnesar — APPROVED** [review]
Reviewed and approved!

**Pull request merged**